README.md

```diff
@@ -30,11 +30,19 @@ To create a hash chain, the arguments are:
 ```shell
 $ ./hashchain create ALGORITHM LENGTH SEED
 ```
+or 
+```shell
+$ ./hashchain create ALGORITHM INDEX SIZE SEED
```

Comment:
I am afraid of the  confusion caused by variant argument counts and sequence of given values. Can we introduce getopt to parse something like `-a ALGORITHM`, `-i INDEX`, `-s SIZE`, etc.?